lib/di.dart

```diff
@@ -248,6 +249,7 @@ Future setup(
       fiatConvertationStore: getIt.get<FiatConversionStore>()));
 
   getIt.registerFactory(() => DashboardViewModel(
+    
```

Comment:
Is that single empty line is necessary ? if it's not necessary please remove it

Reply:
Done